ServerCore/Pages/Resources/ToolsPartial.cshtml

```diff
@@ -1,49 +1,68 @@
-    <p>TEA (The Electronic Alveary) allows you to search word lists for words that match a pattern. TEA's pattern language supports simple template and anagram searches, but also has the flexibility to perform many more complicated queries.  Check the Quick Reference page on the help menu to see the kinds of things TEA can do for you. Typing ;STIN for example will product all anagrams of the letters STIN (ISN'T, NITS, TINS, SNIT).</p>
+    <h3>Types of Puzzles</h3>
+    <p>One of the best ways to learn how to solve puzzles is just to familiarize yourself with all the different kinds of puzzles there are. For example, Japan is famous for its many types of number puzzles.  You've likely heard of Sudoku, but there are tons more!  You can explore and play samples of many of them at sites like <a href="https://www.puzzazz.com/how-to-solve" target="_blank">Puzzazz</a>, <a href="https://www.nikoli.co.jp/en/puzzles/" target="_blank">Nikoli</a>, and <a href="https://www.chiark.greenend.org.uk/~sgtatham/puzzles/" target="_blank">Simon Tatham's</a>. As you can see, some puzzle types even go by multiple names, and there are often types of puzzles that you didn't even know had a name!  Those groups of pictures that represent a word? That's called a "Rebus".</p>
+    <p>Similarly, board and card game mechanics are often used in puzzles, and the solver is expected to know the mechanic by name. Chess moves, Poker hands, and more are all fair game, and this is especially true when the game itself has puzzle-like mechanics.  For example, Boggle works just as well online as it does with the physical cubes!</p>
 
-    <h3><a href="https://nutrimatic.org/">Nutrimatic</a></h3>
-    <p>Nutrimatic is another powerful anagram solver that's backed by Wikipedia's database, so modern words are likely to be found.</p>
+    <h3>Wikipedia</h3>
+    <p>Lots of puzzles deal with manipulating large amounts of manipulation; sometimes it's given to you, sometimes it's up to you to go get it.  Wikipedia is great for this, especially when it comes to large lists of information.  Even small lists are very common for puzzle mechanics:  months of the year, <a href="https://en.wikipedia.org/wiki/Lists_of_constellations" target="_blank">constellations in the sky</a>, US Presidents, basically anything you could think of! It may also help to move that information into a program like Excel that can keep all the columns lined up nicely.</p>
 
-    <h3><a href="https://www.quinapalus.com/qat.html">Qat</a></h3>
-    <p>Qat is a more modern analog of TEA.</p>
+    <br />
+    <h2>Text-Based Puzzles</h2>
+    <h3>Indexing</h3>
+    <p>Perhaps all you're given is a paragraph or a list of words.  If there are numbers associated with them, the first thing to try is to pull out the letter at whichever position in the word an associated number tells you to.  Other times, a number may be telling you how long the word or words you're trying to guess will be.</p>
```

Comment:
Usually you'd solve each item and then index - this sounds like you are indexing into the original content.